README.md

```diff
@@ -142,4 +143,4 @@ Yes, PyAirbyte can use any local install that has a CLI - and will automatically
 
 ## Changelog and Release Notes
 
-For a version history and list of all changes, please see our [GitHub Releases](https://github.com/airbytehq/PyAirbyte/releases) page.
+For a version history and list of all changes, please see our [GitHub Releases](https://github.com/airbytehq/PyAirbyte/releases) page.
```

Comment:
Every file should end with a newline. @ellipsis-dev - Can you please bring back the terminating line feed here?